frontend/src/components/Table/Table.tsx

```diff
@@ -88,8 +99,8 @@ const getStyles = <T extends object>(
     props,
     {
       style: {
-        justifyContent: column?.align === 'right' ? 'flex-end' : 'flex-start',
-        textAlign: column?.align === 'right' ? 'right' : 'left',
+        justifyContent: getJustify(column.align),
```

Comment:
this should respect the existing 'left by default' logic.

Reply:
Potential NPE with **`column.align`** - please use **`column?.align`** instead